test/014-request.test.js

```diff
@@ -5,9 +5,12 @@ import server from '../app';
 
 chai.use(chaiHttp);
 
-const { Request } = models;
+const { Request, User } = models;
 const baseUrl = '/api/v1/user/request';
 let token;
+let token2;
+let id;
+const fakeid = 'BBECD162-0754-44CB-9974-5725E7EA7A94';
```

Comment:
Make the fakeid camel cased please. Also, even though we won't need token2 again, next time, please make the name of any variable more descriptive. For instance, token2 can be `adminToken`. It is clear to who reads the code that a new token was assigned after the user became an admin.

Reply:
ok...